java/ql/src/utils/model-generator/CaptureSummaryModelsSpecific.qll

```diff
@@ -10,7 +10,10 @@ import semmle.code.java.dataflow.internal.DataFlowPrivate
 import semmle.code.java.dataflow.InstanceAccess
 import ModelGeneratorUtils
 
-predicate isOwnInstanceAccess(ReturnStmt rtn) { rtn.getResult().(ThisAccess).isOwnInstanceAccess() }
+pragma[inline]
```

Comment:
No need to inline

Reply:
This still needs to be addressed (and same for the Java file), and then we are good to merge.